docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,174 @@
+
+Your main heading should be H1 and indicate the main content. The incorrect example displays the main heading as H2, which you should only use for subtopics. In addition, don't skip or alter headings as it could cause a fragmented experience, as shown above. H3 is out of order and should follow H2. The correct structure displayed above on the right shows heading levels from h1 to h3 communicating more specific information about the previous section.
+
+For an example video on how to create text headings, see [How Headings Help Screen Reader Users](https://www.youtube.com/watch?v=zRTqCGJ_HwQ).
+
+### Text size and alignment
+The use of white space around blocks of text makes it easier to read. We recommend the following text formatting:
+* left-aligned text
+* font size of 14px
+* 55-65 characters per line
+* sans serif fonts
+
+The following examples demonstrate the incorrect and correct usage of text size and alignment. The incorrect example shows 25-40 characters per line, uses Comic Sans MS, and the font is 11px. The correct example uses the recommended font size of 14 px, has 55-65 characters per line, and uses Arial font.
+
+The correct example demonstrates how to make text readable. To learn more about text size and alignment needed for accessibility, see the [WebAIM article on Typefaces and Fonts](https://webaim.org/techniques/textlayout/).
+
+![Incorrect text](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_incorrect_type_size.png "Incorrect text")
+
+![Correct text](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_correct_text_size.png "Correct type")
+
+### Text links
+Text links should be unique within a page and meaningful when read out of context.
+
+The following example demonstrates how you can make link text meaningful.
+
+| No information | Meaningful information |
+| -- | --|
+| For more information on how to modify the login form, <ins>click here</ins>. | Read more about <ins>modifying the login form</ins>.|
+
+Don't use "click here" for the linked text, instead use the exact title of a linked page or use a description of the linked page. You may need to rework a sentence to make the text meaningful.
+
+Additional linked text recommendations:
+* Use unique link text where possible.
+* Avoid using a URL as the link text, consider using the page title or description of the page.
+* Don't use the word "link" in your link text.
+* Don't capitalize links.
+* Keep link text concise.
+* Don't link directly to downloads.
+* Provide a warning when link text opens a new window.
+* Remove broken or empty links.
+
+## Keyboard accessibility
+Keyboard accessibility involves making a theme accessible without the use of a mouse. Ensure you can navigate to all links, menus, buttons, and form fields using the keyboard. Press the Tab key and navigate from the top of the page to the bottom. The navigation order must be logical and intuitive.
```

Comment:
Adding clarification to sudden subject shift.
Use code highlighting to indicate keyboard keys per style guide: [Pressing and typing keyboard keys](https://intranet.bigcommerce.com/display/SST/UI+Elements+and+Interactions#UIElementsandInteractions-Pressingandtypingkeyboardkeys)
```suggestion
Keyboard accessibility involves making a theme accessible without the use of a mouse. Ensure you can navigate to all links, menus, buttons, and form fields using the keyboard. 

To test a page's keyboard accessibility, press the `Tab` key and navigate from the top of the page to the bottom. The navigation order must be logical and intuitive.
```